.gitattributes

```diff
@@ -0,0 +1,8 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php_cs.dist export-ignore
+/.phpstan.neon export-ignore
+/.styleci.yml export-ignore
+/.travis.yml export-ignore
+/ChangeLog export-ignore
```

Comment:
Please exclude the ChangeLog as I want it to be shipped in any case. Thank you!

Reply:
Done, wasn't sure about this.